x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -246,23 +247,24 @@ export const signalRulesAlertType = ({
               name,
               id: savedObject.id,
             };
-            const { signalsCount, resultsLink } = await getSignalsCount({
-              from: `now-${interval}`,
-              to: 'now',
-              index: ruleParams.outputIndex,
-              ruleId: ruleParams.ruleId!,
+
+            const { fromInMs, toInMs } = parseDateRanges(`now-${interval}`, 'now');
+            const resultsLink = getNotificationResultsLink({
+              from: fromInMs,
+              to: toInMs,
+              id: savedObject.id,
               kibanaSiemAppUrl: meta?.kibanaSiemAppUrl as string,
-              ruleAlertId: savedObject.id,
-              callCluster: services.callCluster,
             });
 
-            logger.info(buildRuleMessage(`Found ${signalsCount} signals for notification.`));
+            logger.info(
+              buildRuleMessage(`Found ${result.createdSignalsCount} signals for notification.`)
+            );
 
-            if (signalsCount) {
+            if (result.createdSignalsCount) {
               const alertInstance = services.alertInstanceFactory(alertId);
               scheduleNotificationActions({
                 alertInstance,
-                signalsCount,
+                signalsCount: `${result.createdSignalsCount}`,
```

Comment:
Can `signalsCount` be a number instead of a string?

Reply:
Fixed